### DISCUSSION TITLE
Hello I am trying to make a calendar can you help me

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#73] [Q&A] [answered]
I am trying to put my own desired lunar months. Can you give me the code of that so that it could work please.

hmUI.createWidget(hmUI.widget.TEXT, {
							x: 20,
							y: 400,
							w: 180,
							h: 100,
							color: 0x00CACA,	
							text_size: 25,
							text_style: hmUI.text_style.WRAP,
							text: time.lunar_month
							});

**Comment by leober-ramos33** [comment, marked as answer]
Here is code that can help you: https://github.com/melianmiko/SmartBand7-Apps/tree/master/calendar

**Reply by alexmathew11** [reply]
Thanks
